app/interfaces/api/entities/search_result.rb

```diff
@@ -60,31 +60,31 @@ def redetermination
       end
 
       def fixed_fee
-        object.is_fixed_fee.eql?('t')
+        object.is_fixed_fee.eql?('t') && is_submitted?
       end
 
       def awaiting_written_reasons
         object.state.eql?('awaiting_written_reasons')
       end
 
       def cracked
-        object.case_type.eql?('Cracked Trial')
+        ['Cracked before retrial', 'Cracked Trial'].include?(object.case_type) && is_submitted?
       end
 
       def trial
-        object.case_type.eql?('Trial')
+        %w[Trial Retrial].include?(object.case_type) && is_submitted?
       end
 
       def guilty_plea
-        object.case_type.eql?('Guilty plea')
+        ['Discontinuance', 'Guilty plea'].include?(object.case_type) && is_submitted?
       end
 
       def graduated_fees
-        object.fee_type_code&.in?(graduated_fee_codes).eql?(true)
+        (object.fee_type_code&.in?(graduated_fee_codes).eql?(true) || allocation_type_is_grad?) && is_submitted?
```

Comment:
I think it can be used like that... refactoring is due when we get to re-allocation! :)